models/chain-information.ts

```diff
@@ -0,0 +1,21 @@
+export interface ChainInformation {
+  chainName: string
+  description: string
+  platformChainId: string
+  subnetId: string
+  vmId: string
+  vmName: string // EVM, BITCOIN, ETHEREUM
+  rpcUrl: string // uri
+  wsUrl?: string // uri
+  multicallAddress?: string // ChecksumAddress
+  isTestnet: boolean
+  networkToken: {
+    name: string
+    symbol: string
+    description: string
+    decimals: number
+    coingeckoId?: string
+  }
+  coingeckoAssetPlatformId?: string
+  chainId: number
```

Comment:
nit: maybe define chainId as the first property since it is (probably) the most important.

Reply:
Agreed 👍